src/basic/FStar.Options.fs

```diff
@@ -1369,8 +1369,9 @@ let parse_settings ns : list<(list<string> * bool)> =
              (path_of_text s, true)
     in
     ns |> List.collect (fun s ->
-          FStar.Util.split s " "
-          |> List.map parse_one_setting)
+          if s = "" then []
+          else FStar.Util.split s " "
+               |> List.map parse_one_setting)
```

Comment:
This function may be badly merged. On master, I added some caching of parse_one_setting.

See https://github.com/FStarLang/FStar/blob/master/src/basic/FStar.Options.fs#L1360